pkgs/development/python-modules/torch/fix-cmake-cuda-toolkit.patch

```diff
@@ -1,20 +1,20 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 9194e520bb0..d05fdcfb6cb 100644
+index c4cd4b2c2..e983b2135 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -1160,10 +1160,6 @@ if(BUILD_SHARED_LIBS)
-       ${PROJECT_SOURCE_DIR}/cmake/Modules_CUDA_fix
-       DESTINATION share/cmake/Caffe2/
-       COMPONENT dev)
--  install(FILES
--      ${PROJECT_SOURCE_DIR}/cmake/Modules/FindCUDAToolkit.cmake
--      DESTINATION share/cmake/Caffe2/
--      COMPONENT dev)
-   install(FILES
-       ${PROJECT_SOURCE_DIR}/cmake/Modules/FindCUSPARSELT.cmake
-       DESTINATION share/cmake/Caffe2/
+@@ -1319,10 +1319,6 @@ if(BUILD_SHARED_LIBS)
+     DIRECTORY ${PROJECT_SOURCE_DIR}/cmake/Modules_CUDA_fix
+     DESTINATION share/cmake/Caffe2/
+     COMPONENT dev)
+-  install(
+-    FILES ${PROJECT_SOURCE_DIR}/cmake/Modules/FindCUDAToolkit.cmake
+-    DESTINATION share/cmake/Caffe2/
+-    COMPONENT dev)
+   install(
+     FILES ${PROJECT_SOURCE_DIR}/cmake/Modules/FindCUSPARSELT.cmake
+     DESTINATION share/cmake/Caffe2/
```

Comment:
THat said, I do not think we want to be installing any of the vendored cmake files. It's strange we're not deleting findcusparselt.cmake (doesn't have to be addressed in this pr)